dlgr/griduniverse/experiment.py

```diff
@@ -303,6 +307,19 @@ def __init__(self, **kwargs):
             self.color_costs = [2**i for i in range(self.num_colors)]
             random.shuffle(self.color_costs)
 
+        # get food spawning probability distribution function and args
+        self.probability_function_args = []
```

Comment:
We need some error catching for incorrect keys entered as probability distributions.